server/routes/api/reactions/reactions.ts

```diff
@@ -0,0 +1,45 @@
+import Router from "koa-router";
+import auth from "@server/middlewares/authentication";
+import validate from "@server/middlewares/validate";
+import { Comment, Reaction } from "@server/models";
+import { authorize } from "@server/policies";
+import { presentReaction } from "@server/presenters";
+import { APIContext } from "@server/types";
+import * as T from "./schema";
+
+const router = new Router();
+
+router.post(
+  "reactions.list",
+  auth(),
+  validate(T.ReactionsListSchema),
+  async (ctx: APIContext<T.ReactionsListReq>) => {
+    const { commentId } = ctx.input.body;
+    const { user } = ctx.state.auth;
+
+    const comment = await Comment.findByPk(commentId, {
+      rejectOnEmpty: true,
+    });
+
+    authorize(user, "readReaction", comment);
```

Comment:
Lets also check you have permission to read the associated document here, as here:

https://github.com/outline/outline/blob/d3ddb25c760aeb29120d8d1284e6f40b81e7e731/server/routes/api/comments/comments.ts#L73